source/content/guides/build-tools/01-introduction.md

```diff
@@ -19,6 +19,13 @@ editpath: build-tools/01-introduction.md
 image: buildToolsGuide-thumb
 multidev: true
 ---
+
+<Alert type="danger" title="Warning">
+
+The current version of the Build Tools plugin is designed for Terminus 1.x, and is not yet comptaible with [current versions](/terminus/updates/) of Terminus. We'll update this guide when the plugin is officially updated, or you can try the beta release (unsupported) [here]().
```

Comment:
@alexfornuto 

Can you change `We'll update this guide when the plugin is officially updated, or you can try the beta release` to `We are working on a new version of the Terminus Build Tools plugin with Terminus 2 support and will update this guide once it is released. In the meantime, you can test the beta release.`

Reply:
The beta link is https://github.com/pantheon-systems/terminus-build-tools-plugin/releases/latest

Reply:
[Updated](a1dd26dc81e6fd97ad8dc3d0f8623805eafb5cee)